xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -39,9 +39,20 @@
         <Parameter Name="allowedCipherSuites" Type="System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;" />
       </Parameters>
       <Docs>
-        <param name="allowedCipherSuites">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="allowedCipherSuites">The collection of cipher suites allowed in this policy for negotiation.</param>
+        <summary>Initializes a new instance of the <see cref="T:System.Net.Security.CipherSuitesPolicy" /> class that can be used on a client or server authentication.</summary>
```

Comment:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Net.Security.CipherSuitesPolicy" /> class that can be used on a client or server for authentication.</summary>
```

Reply:
I don't think `for` is missing here, since I am trying to refer to `client authentication` and `server authentication` mechanisms.

Reply:
In that case, "for client or server authentication" might be better.

Reply:
All right, that sounds good to me.

```suggestion
        <summary>Initializes a new instance of the <see cref="T:System.Net.Security.CipherSuitesPolicy" /> class that can be used for client or server authentication.</summary>
```